tests/eligibility/factories.py

```diff
@@ -122,6 +122,9 @@ class Params:
             certified=False,
             certified_at=factory.SelfAttribute(".eligibility_diagnosis.created_at"),
         )
+        criteria_certification_error = factory.Trait(
+            certified_at=factory.SelfAttribute(".eligibility_diagnosis.created_at"),
+        )
```

Comment:
_nit_, je trouve ça plus clair en arrivant dessus sinon on se demande un peu pourquoi juste avoir `certified_at` est une erreur 😅.
```suggestion
        criteria_certification_error = factory.Trait(
            certified=None,
            certified_at=factory.SelfAttribute(".eligibility_diagnosis.created_at"),
        )
```